themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Introducing Azure Native 2.0"
+
+date: 2023-06-14T14:00:00-04:00
```

Comment:
Needs a bump to today

Reply:
This is the planned time for today. 6/14 at 2pm ET, 11AM PT